assets/less/appeals/dec24.less

```
@@ -0,0 +1,397 @@
+    line-height: 1.6;
+  }
+
+  .overlay {
+    position: absolute;
+    pointer-events: none;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    opacity: 0.2;
+    background: var(--overlay-bg);
+
+    // Figma creates *special* gradients
+    transform: rotateZ(180deg) scaleX(-1);
+  }
+
+  #bg-img {
+    // Prefer webp over the png
+    --bg-img: url('/media/img/thunderbird/appeal/dec24/background-image-high-res.png');
+    --bg-img-set: image-set(var(--bg-img) type('image/png'),
+    url('/media/img/thunderbird/appeal/dec24/background-image-high-res.webp') type('image/webp'));
+    --bg-size: 1091px 1008px;
+    --bg-position: center top;
+
+    position: absolute;
+    top: 0;
+    display: flex;
+    width: 100%;
+    height: 100%;
+    max-height: 500px;
+    justify-content: center;
+    z-index: -1;
+
+    background-image: var(--bg-img);
+    background-image: var(--bg-img-set);
+    background-position: center top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  #appeal-header {
+
+    position: relative;
+    /* left: 50px; */
+
+    display: flex;
+    flex-direction: column-reverse;
+    gap: 30px;
+    margin: 0 auto 52px;
+    padding: 0 24px 24px;
+
+    width: 100%;
+    max-width: 900px;
+    align-items: center;
+    font-size: 1.25rem;
+
+    @media (min-width: @ss) {
+      gap: 60px;
+    }
+
+    @media (min-width: @sm) {
+      gap: 60px;
+      padding: 0;
+    }
+
+    @media (min-width: @md) {
+      padding: 0 24px 24px;
+      flex-direction: row;
+      gap: 9px;
+      align-items: flex-end;
+    }
+
+
+    & header {
+
+      /* Disable inherited styles */
+      background: none;
+      padding-top: 0;
+      padding-bottom: 0;
+      position: relative;
+
+      // todo: find a better way to set height?
+      height: 10rem;
+
+      @media (min-width: @md) {
+        height: 15rem;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-end;
+      }
+    }
+
+
+    #appeal-heading {
+      --txt: #FEFFFF;
+      width: 100%;
+      text-align: left;
+      display: flex;
+      flex-direction: column;
+
+      text-transform: uppercase;
+      font-size: 2.0rem;
+      font-weight: 600;
+      color: var(--txt);
+
+      & > span {
+        display: block;
+        font-size: 3.5rem;
+        font-weight: 600;
+        line-height: 3.5rem;
+
+
+
+      }
+    }
+
+    #illustration {
+      width: 90%;
+      height: 10rem;
+      position: relative;
+      @media (min-width: @md) {
+        width: 80%;
+      }
+    }
+    #illustration > #roc {
+      position: absolute;
+      top: -100px;
+
+      @media (min-width: @ss) {
+        top: -120px;
+      }
+      @media (min-width: @sm) {
+        top: -180px;
+      }
+      @media (min-width: @md) {
+        top: -250px;
+        left: -69px;
+
+      }
+    }
+
+  }
+
+  main {
+    padding: 1em;
+    margin: 0 auto;
+    max-width: 787px;
```

This has been changed to 964.18px, but can you swap it to rems. 
```suggestion
    max-width: 60.26125rem;
```